portfolio/src/main/webapp/script.js

```diff
@@ -88,4 +154,35 @@ function createParagraphElement(text) {
   const pElement = document.createElement('p');
   pElement.innerText = text;
   return pElement;
+}
+
+/** Creates a combination of HTML elements representing a comment */
+function createComment(commentData) {
```

Comment:
The name commentData refers to single comment or collection of comments in different places in the file. This could be confusing.